packages/node_modules/@ciscospark/react-component-people-list/src/index.js

```diff
@@ -16,56 +19,99 @@ const propTypes = {
       isPending: PropTypes.bool
     }))
   })).isRequired,
-  onItemClick: PropTypes.func
+  onItemClick: PropTypes.func,
+  onDisplayUsers: PropTypes.func
 };
 
 const defaultProps = {
-  onItemClick: () => {}
+  onItemClick: () => {},
+  onDisplayUsers: () => {}
 };
 
-
 function PeopleList({
   items,
-  onItemClick
+  onItemClick,
+  onDisplayUsers
 }) {
   if (!items || !items.length) {
     return null;
   }
 
-  function generateGroupList(people) {
-    return people.map((p) => {
+  const totalRows = [];
+
+  items.forEach((item) => {
+    // Header for item
+    if (item.label) {
+      totalRows.push({type: 'header', details: item.label});
+    }
+    // Rows for each person
+    item.people.forEach((person) => {
+      totalRows.push({type: 'person', details: person});
+    });
+  });
+
+  function rowRenderer(options) {
+    const {
+      key,
+      index,
+      style
+    } = options; {
```

Comment:
This should probably be destructured in the function param like this:
```js
function rowRenderer({key, index, style}) {
```

Reply:
FYI, this is a copy pasta from my code which was a copy pasta from virtualized demo docs. Is there an eslint rule we can add for these?